frappe/public/js/frappe/form/footer/attachments.js

```diff
@@ -13,6 +13,9 @@ frappe.ui.form.Attachments = Class.extend({
 		this.parent.find(".add-attachment").click(function() {
 			me.new_attachment();
 		});
+		this.parent.find(".add-gsuite").click(function() {
```

Comment:
This should be in the attachment popup, we can directly have an option to upload to GSuite or any other service in a generic way. Can you do that?